### PR TITLE
core-49 add analysis history to analysis info dialog.

### PR DESCRIPTION
**Pull request by sri2k1us**
image

### CODE REVIEW

File: react-components/src/analysis/ids.js [hunk 1]
```diff
@@ -1,63 +1,70 @@
...
-    OWNER: "owner",
-    APP: "app",
-    START_DATE: "start_date",
-    END_DATE: "end_date",
+export default {
+  CHECKBOX: ".checkbox",
+  NAME: "name",
+  OWNER: "owner",
+  APP: "app",
+  START_DATE: "start_date",
```
Comment (ash3rz) [inline react-components/src/analysis/ids.js]: Just picking a random spot - did the indentation change from 4 spaces to 2?  It seems like all the *.js files in this PR now have tighter spacing.

Comment (sri2k1us) [reply react-components/src/analysis/ids.js]: I think IntelliJ applied some Prettier formatting 😄. I will rebase on `master` and format it one more time just to be sure.

File: react-components/src/analysis/view/dialogs/AnalysisInfoDialog.js [hunk 2]
```diff
@@ -1,75 +1,144 @@
...
+              </TableCell>
+              <TableCell className={classes.analysisInfoFont}>
+                {update.message}
+              </TableCell>
+              <TableCell className={classes.analysisInfoFont}>
+                {status[0].toUpperCase() +
+                  status
+                    .slice(1)
+                    .toLowerCase()
+                    .replace(/[_]/gi, " ")}
```
[the remark this answers is not in the document]
Comment (sri2k1us) [reply react-components/src/analysis/view/dialogs/AnalysisInfoDialog.js]: In this case, I want a `sentence` case.

File: react-components/stories/analysis/view/dialogs/AnalysisInfoDialog.stories.js [hunk 3]
```diff
@@ -1,26 +1,193 @@
...
+            },
+            {
+              status: "FINISHED",
+              message: "Job complete",
+              timestamp: "1553818416000"
+            }
+          ]
+        },
+        {
+          step_number: 2,
```
Comment (psarando) [inline react-components/stories/analysis/view/dialogs/AnalysisInfoDialog.stories.js]: 👍

**Comment by sri2k1us** [comment]
Thanks for the reviews.

**Pull request merged**